lib/pipeline.js

```
@@ -135,6 +135,33 @@ Pipeline.prototype = {
   },
 
   /**
+   * Bind to a streams events, and re-emit them on the pipeline
+   * @param {Stream} stream
+   * @param {String|Array} events
+   * @return {Boolean} bound
+   */
+  bind: function( stream, events ) {
+
+    if( this.indexOf( stream ) === -1 )
+      return false
```

Should this raise an error, rather than simply returning `false`?